genesyscloud/telephony_providers_edges_site/resource_genesyscloud_telephony_providers_edges_site_test.go

```diff
@@ -19,6 +20,20 @@ import (
 	"github.com/mypurecloud/platform-client-sdk-go/v129/platformclientv2"
 )
 
+func assignregion() string {
```

Comment:
We will most likely need to use this function in a few tests, you can move it into `test_utils.go` and import it where needed

Reply:
Moved it to test_utils.go